roles/imagemagick/vars/Ubuntu.yml

```diff
@@ -0,0 +1,9 @@
+---
+im_packages:
+  - autoconf # already present
+  - automake # already present
+  - cmake # missing - move this to prerequisites?
```

Comment:
If we move installing cmake out of the role, then this role won't work outside of a Hydra playbook. My instinct says to make some choices about the structure and make them explicit - see https://github.com/curationexperts/ansible-hydra/issues/158. That said, I'm open to consolidating all prereqs in one place and saying the ansible-hydra roles only work together.